src/esplog/mini_decompressor.rs

```diff
@@ -0,0 +1,691 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use std::fmt::{Debug, Display};
+use std::ops::{Add, Div, Mul, Neg, Sub};
+
+#[derive(Copy, Clone, PartialOrd, PartialEq, Ord, Eq)]
+pub struct Fix32<const N: usize> {
+    v: i32,
+}
+
+impl<const N: usize> Fix32<N> {
```

Comment:
Did you consider using the [`fixed`](https://docs.rs/fixed/latest/fixed/) crate instead of own implementation?

Reply:
Okay, I decided to get rid of quats and fixed numbers at the decoder.

The initial idea was to make it behave exactly like the implementation in the encoder so the encoder exactly knows the state in which the decoder will be at any time and knows the absolute angular error. 

But as I do not actually need the quaternions at the decoder (I convert them back to angular velocity anyway), I can get rid of them.